src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import classNames from 'classnames';
+
+type Props = {
+  isVisible: boolean,
+  errorText?: null | string,
```

Comment:
The `errorText` prop should be required. As it stands, it's possible to show an error message without any text. To improve the code, the `errorText` prop shouldn't be optional and shouldn't allow null values.